packages/bot/src/stores/run-panel-store.js

```diff
@@ -72,25 +65,17 @@ export default class RunPanelStore {
 
         this.registerBotListeners();
 
-        if (!hasAllRequiredBlocks()) {
```

Comment:
@aaron-binary you removed the required block check from here, are you handling it somewhere else?  couldn't find it :)

Reply:
I've moved it to `scratch/index.js`, please see line 214 or search for `checkForRequiredBlocks`.